content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,354 @@
+----------------------
+
+To configure your company information, go to the :menuselection:`Settings --> Companies: Update
+Info` and configure the following:
+
+- :guilabel:`Company Name`
+- :guilabel:`Address`:
+
+  - :guilabel:`Street`
+  - :guilabel:`City`
+  - :guilabel:`State`
+  - :guilabel:`ZIP`
+  - :guilabel:`Country`
+
+- :guilabel:`Tax ID`: Enter the identification number for the selected taxpayer type.
+- :guilabel:`DGI Main Branch Code`: This is part of the XML when creating an electronic document. If
+  this field is not set, all electronic documents will be rejected.
+
+  To find the *DGI Main Branch Code*, follow these steps:
+
+  #. From your :ref:`Uruware account <l10n_uy/uruware-account>`, go to :menuselection:`Servicios en
+     línea DGI --> Registro único tributario --> Consulta de datos`.
+  #. Then, select :menuselection:`Consulta de Datos Registrales --> Consulta de Datos de Entidades`.
+  #. Next, open the generated PDF to get the *DGI Main Branch Code* from the :guilabel:`Domicilio
+     Fiscal Número de Local` section.
+
+After configuring the company in the database settings, navigating to :menuselection:`Contacts` and
+search for your company to verify the following:
+
+- Company type is set to :guilabel:`Company`.
+- Verify the :guilabel:`Identification Number` :guilabel:`Type` is :guilabel:`RUT / RUC`.
+
+.. _l10n_uy/uruware-account:
+
+Set up Uruware account
+----------------------
+
+To generate electronic documents in Odoo, a Uruware account is necessary.
+
+To set up a Uruware account, first be sure to have a valid Odoo subscription. Then follow the
+proceeding steps:
+
+Locate the Uruware credentials settings by navigating to the :menuselection:`Accounting -->
+Configurations --> Settings --> Localización Uruguaya`. Take a note of the admin email address that
+is set, this will be the same email used in Uruware's portal. Click on :guilabel:`Create Uruware
+Account`.
+
+.. tip::
+   Enter the email of the administrator or project leader. Avoid using the same email as the one
+   used for communication with the |DGI|.
+
+.. note::
+   This action will create an account with Uruware with the following information:
+
+   - Legal name (razón social)
+   - RUT from the company
+   - Administrator email
+   - User name and last name
+   - Odoo database link
+
+   To ensure your account is created correctly, please add any missing information from above.
+
+Once the account is created, a confirmation email is sent to the email used to create the account,
+containing the account credentials to configure the account directly in the Uruware portal:
+
+- Testing portal: https://odootest.ucfe.com.uy/Gestion/
+- Production portal: https://prod6109.ucfe.com.uy/Gestion/
+
+Use the account credentials in the email to log in to the portal.
+
+.. important::
+   Be sure to configure a testing account and a production account in the Uruware portal. The same
+   Uruware account can be used for both environments.
```

Comment:
```suggestion
   Be sure to configure two accounts, one for testing and one for production. The email address 
   can be changed in Uruware and the same can be used in both environments.
```

Reply:
https://github.com/odoo/documentation/pull/9849#discussion_r1715487691 implemented in 7d28b87